### PR TITLE
Change encryption/decryption algorithm to A256CBC

### PR DESCRIPTION
**Pull request by ganeshamohana**

### CODE REVIEW

**Review by SethBe — DISMISSED** [review]
:shipit:

**Comment by SethBe** [comment]
LOoks good to me from a Mocwrapper perspective. Please share the bins when ready and i can do some smoke testing.

File: services/security/keyvault/key/key.go [hunk 1]
```diff
@@ -257,6 +257,8 @@ func getMOCAlgorithm(algo keyvault.JSONWebKeyEncryptionAlgorithm) (wssdcloudcomm
 		return wssdcloudcommon.Algorithm_RSAOAEP256, nil
 	case keyvault.A256KW:
```
Comment (raghavendra-nataraj) [inline services/security/keyvault/key/key.go:258]: Are we still supporting this?

Comment (ganeshamohana) [reply services/security/keyvault/key/key.go:258]: No, we'll reject the request for this.

Comment (raghavendra-nataraj) [reply services/security/keyvault/key/key.go:258]: Is this used by someother api call, if not we can remove it

**Review by raghavendra-nataraj — APPROVED** [review]

**Review by SethBe — APPROVED** [review]
:shipit:

**Pull request merged**